src/utils/topics/removePrefixTopic.js

```diff
@@ -1,15 +1,15 @@
-import { PREFIX_TOPIC } from "../constants"
-import { convertString } from "../string/StringUtils"
+import {PREFIX_TOPIC} from '../constants';
+import {convertString} from '../string/StringUtils';
 
-const removePrefixTopic = topicWithPrefix => {
-    if(topicWithPrefix === null || topicWithPrefix === undefined) return
+const removePrefixTopic = (topicWithPrefix) => {
+  if (topicWithPrefix === null || topicWithPrefix === undefined) return;
 
-    if (topicWithPrefix.indexOf(PREFIX_TOPIC) > -1) {
-        const topic = convertString(topicWithPrefix, PREFIX_TOPIC, '');
-        return topic;
-    }
+  if (topicWithPrefix.indexOf(PREFIX_TOPIC) > -1) {
+    const topic = convertString(topicWithPrefix, PREFIX_TOPIC, '');
+    return topic;
+  }
 
-    return topicWithPrefix
-}
+  return topicWithPrefix;
+};
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There should be a newline at the end of the file. This is a common convention that makes it easier to concatenate files and helps certain tools process your files more correctly.
```diff
- \ No newline at end of file
+ 
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->